packages/client/webapp/src/components/ProjectCard.vue

```diff
@@ -0,0 +1,134 @@
+<script setup lang="ts">
+// import Button from '@/components/Button.vue';
+import IconList24 from '@carbon/icons-vue/es/list/24';
+import IconAddFilled32 from '@carbon/icons-vue/es/add--filled/32';
+
+interface Props {
+	category: string;
+	projectName: string;
+	isNewProjectCard: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+	category: '',
+	projectName: '',
+	isNewProjectCard: false
+});
+
+function createNewProject() {}
+</script>
+
+<template>
+	<div v-if="!props.isNewProjectCard" class="project-card">
+		<div class="details">
+			<div class="name">{{ props?.projectName }} {{ props?.p }}</div>
+			<div class="actions">
+				<IconList24 />
+			</div>
+		</div>
+	</div>
+	<div v-else class="new-project-card">
+		<IconAddFilled32 class="icon-add-filled" @onclick="createNewProject" />
+		<div class="details">
+			<div class="create-new-project">Create New Project</div>
+		</div>
+	</div>
+	<!-- <div v-else class="new-project-card">
+        <Button @onclick="createNewProject">Create New Project
+            <IconAddFilled32 />
+        </Button>
+    </div> -->
+</template>
+
+<style scoped>
+.project-card,
+.new-project-card {
+	color: var(--un-color-black-80);
+	border: 1px solid var(--un-color-black-20);
+	background-color: var(--un-color-black-5);
+	font-weight: 500;
```

Comment:
- You can remove the `color` and `font-weight`, better keep it default
- for the background colour please use `--un-color-body-surface-primary`